packages/react/migrations.json

```diff
@@ -104,6 +104,11 @@
       "version": "13.0.0-beta.0",
       "description": "Migrate Storybook to use webpack 5",
       "factory": "./src/migrations/update-13-0-0/migrate-storybook-to-webpack-5"
+    },
+    "update-external-emotion-jsx-runtime-13.3.0": {
+      "version": "13.3.0-beta.0",
+      "description": "Update external option in projects for Emotion",
+      "factory": "./src/migrations/update-13-3-0/update-external-emotion-jsx-runtime"
```

Comment:
I think I need @jaysoo 's advice which future version of Nx should I apply this migration.